local-dev/run_postgres.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# The CloudSQL console simply states "PostgreSQL 9.6" so we may not match the minor version number
+POSTGRES_VERSION=12.3
```

Comment:
9.6 and 12.3 seem like more than minor major version number.

Reply:
9.6 is too old. The verson on GCP is 12 now. So I just choose the latest verison.

Reply:
Maybe just remove the comment on line 3 then?

Reply:
done